addons/binding/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/KebaBindingConstants.java

```diff
@@ -50,5 +50,85 @@
     public static final String CHANNEL_POWER_FACTOR = "powerfactor";
     public static final String CHANNEL_SESSION_CONSUMPTION = "sessionconsumption";
     public static final String CHANNEL_TOTAL_CONSUMPTION = "totalconsumption";
+    public static final String CHANNEL_DISPLAY = "display";
+
+    public enum KebaType {
+        P20,
+        P30
+    };
+
+    public enum KebaSeries {
+        E {
+            @Override
```

Comment:
Enum specific method bodies are not the most friendly alternative, for another time it might be more easy to have centralized logic:

```java
enum Example {
E('a'),
B('a', 'c');

private final List<Character> things;

Example(char... e){
    things = Arrays.asList(e);
}

public boolean matchesSeries(char c) {
    return things.contains(c);
}
```